terraform/modules/tf_lambda/cloudwatch.tf

```diff
@@ -1,5 +1,42 @@
+/* CloudWatch event rules, log group, and metric alarms */
+
+// CloudWatch event to trigger Lambda in a cronjob (if applicable)
+
+resource "aws_cloudwatch_event_rule" "cronjob" {
```

Comment:
Could we change the verbiage to be 'scheduled' instead of 'cronjob'? It may be slightly confusing since CWE supports `rate()` expressions and `cron` based expressions

Reply:
Yeah, good idea